New package: r-hh

glennpj opened the pull request.

glennpj committed:
New package: r-hh

alalazo commented on var/spack/repos/builtin/packages/r-hh/package.py:
```diff
@@ -0,0 +1,44 @@
+
+
+class RHh(RPackage):
+    """Support software for Statistical Analysis and Data Display (Second
+    Edition, Springer, ISBN 978-1-4939-2121-8, 2015) and (First Edition,
```
```suggestion
    """Support software for Statistical Analysis and Data Display.
    
    Support software for Statistical Analysis and Data Display (Second
    Edition, Springer, ISBN 978-1-4939-2121-8, 2015) and (First Edition,
```

glennpj replied:
I did not realize that was parsed that way. I see that first line gets uses by `module whatis`. Which spack commands use the short description?

I will use the one liner from the project page.

Thanks.

adamjstewart replied:
I believe the module files Spack generates use the short description, while `spack info` uses the full description.

alalazo reviewed (dismissed):
LGTM, just a request to add a short first paragraph to the docstring (it will be used where a short description of the package is needed by commands or modules).

glennpj committed:
Add short description

Add the project one liner for a short description that will be used for
`module whatis`.
